sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -969,6 +969,11 @@ def _verify_interpreter_version_is_supported(pipeline_options):
 
 
 # To enable a counter on the service, add it to this dictionary.
+# This is required for the legacy python dataflow runner, as portability
+# does not communicate to the service via python code, but instead via a
+# a runner harness (in C++ or Java).
+# TODO(BEAM-7050) : Remove this antipattern, legacy dataflow python
+# pipelines will break whenever a new cy_combiner type is used.
```

Comment:
to be fair, it currently breaks, but we can handle the failure and log rather than just failing.